planning/technology/storage/README.md

```diff
@@ -66,8 +66,47 @@ This section details the requirements and options specifically for the manuscrip
 
 ##### Potential Partners
 
-1. S3 - storage for git
-2. ElasticSearch - duplicate data to make searchable
+Jack is thinking that really we probably want all of these:
+
+1. git (for versioning)
+2. S3 (for durability)
+3. ElasticSearch (for real-time querying)
+
+If we treat S3 files as immutable, they serve as our disaster-recovery solution
+and easy rollback/restore source. Git repos are considered current 'source of
+truth' versions, unless there's a catastrophe and we manually fall back to an
+s3 snapshot. Elasticsearch is considered ephemeral, and gets completely blown
```

Comment:
My main concern here is the price we'd have to pay to be constantly kicking off the indexer. Might be fun, but we would need to make sure that it's viable at scale.